batch_worker.py

```diff
@@ -140,6 +141,11 @@ def get_cluster_id_batch(self,end_date, batch_size):
         ]
         return result
     
+    def get_grafana_url_for_namespace(self, df):
+        return f"""{df['grafana_url']}/d/{self.grafana_dashboard_uid}/
```

Comment:
@dvandra We could have a check for valid value for df['grafana_url']

Reply:
Added one check for uid that it is not null.